sdk/communication/Azure.Communication.Rooms/tests/RoomsClient/RoomsClientTest.cs

```diff
@@ -20,29 +20,31 @@ public async Task CreateRoomAsyncShouldSucceed()
             var validFrom = DateTime.UtcNow;
             var validUntil = validFrom.AddDays(1);
 
-            List<RoomParticipant> createRoomParticipants = new List<RoomParticipant>();
             var mri1 = "8:acs:1b5cc06b-f352-4571-b1e6-d9b259b7c776_00000007-0464-274b-b274-5a3a0d000101";
             var mri2 = "8:acs:1b5cc06b-f352-4571-b1e6-d9b259b7c776_00000007-0464-274b-b274-5a3a0d000102";
 
             var communicationUser1 = new CommunicationUserIdentifier(mri1);
             var communicationUser2 = new CommunicationUserIdentifier(mri2);
 
-            var participant1 = new RoomParticipant(communicationUser1, RoleType.Presenter);
-            var participant2 = new RoomParticipant(communicationUser2, RoleType.Attendee);
+            var participant1 = new RoomParticipant(communicationUser1) { Role = ParticipantRole.Presenter };
```

Comment:
Just wonder if we have a scenario to test against a default role when Contoso does not specify the role.

Reply:
Yes we do, this file is just for mocked tests. But if you look at RoomsClientLiveTests.cs `RoomParticipantsAddUpdateAndRemoveWithNullRolesLiveTest` we check that exact case you're talking about.